sdk/keyvault/keyvault-keys/migration-guide.md

```diff
@@ -0,0 +1,251 @@
+  const cryptographyClient = new CryptographyClient(keyVaultKey.id!, credential);
+  const signed = await cryptographyClient.signData("RS256", "some data");
+  console.log(signed);
+}
+
+main().catch((err) => console.error(err));
+```
+
+### Create a key
+
+In `azure-keyvault` you could create a key by using `KeyVaultClient`'s `createKey` method, which required a vault endpoint, key name, and key type. This method returned a `KeyBundle` containing the key.
+
+```js
+// create an RSA key
+let keyBundle = await client.createKey(vaultUrl, "myRSAKey", "RSA");
+console.log(keyBundle);
+
+// create an elliptic curve key
+keyBundle = await client.createKey(vaultUrl, "myECKey", "EC")
+console.log(keyBundle);
+```
+
+Now in `@azure/keyvault-keys` there are multiple ways to create keys. You can provide a key name and type to the general `createKey` method, or provide just a name to `createRsaKey` or `createEcKey`. These methods all return the created key as a `KeyVaultKey`.
+
+```ts
+const rsaKey1 = await client.createKey("MyRSAKey1", "RSA");
+const rsaKey2 = await client.createRsaKey("MyRSAKey2");
+const ecKey1 = await client.createKey("MyECKey1", "EC");
+const ecKey2 = await client.createEcKey("MyECKey2");
+```
+
+### Retrieve a key
+
+In `azure-keyvault` you could retrieve a key (in a `KeyBundle`) by using `getKey` and specifying the desired vault endpoint, key name, and key version. You could retrieve the versions of a key with the `getKeyVersions` method, which returned an array with key items that can be used to retrieve the full key afterwards.
+
+```js
+const keyItems = await client.getKeyVersions(vaultUrl, "MyKey");
+
+for (let keyItem of keyItems) {
+  const version = keyItem.kid.split("/").pop();
+  console.log({ version });
+
+  const keyBundle = await client.getKey(vaultUrl, "MyKey", version);
+  console.log({ keyBundle });
+}
+```
+
+Now in `@azure/keyvault-keys` you can retrieve the latest version of a key (as a `KeyVaultKey`) by using `getKey` and providing a key name. In addition, `listPropertiesOfKeyVersions` can be used to iterate over the versions of a specific key.
+
+```ts
+const keyVaultKey = await client.getKey("MyKey");
+console.log(keyVaultKey.name);
+console.log(keyVaultKey.keyType);
+console.log(keyVaultKey.properties.version);
+
+for await (let versionProperties of client.listPropertiesOfKeyVersions("MyKey")) {
+  console.log("Name:", versionProperties.name, "Version:", versionProperties.version);
+  const keyVaultKey = await client.getKey(versionProperties.name, { version: versionProperties.version });
+  console.log(keyVaultKey.properties.version);
+}
+```
+
+### List properties of keys
+
+In `azure-keyvault` you could list the properties of keys in a specified vault with the `getKeys` method. This returned an array containing the basic properties of each available key.
+
+```js
+const keyItems = await client.getKeys(vaultUrl);
+for (let keyItem of keyItems) {
+  console.log(keyItem.kid);
```

Comment:
Here we log the `kid`, in Track 2 code snippet we log the `name`. Can we pick one and log the same in both cases?